lib/config/Configuration.js

```diff
@@ -0,0 +1,91 @@
+import path from "node:path";
+import os from "node:os";
+
+/**
+ *
+ *
+ * @public
+ * @class
+ * @alias @ui5/project/config/Configuration
+ */
+export class Configuration {
+	#snapshotEndpointUrl;
+
+	/**
+	 * @param {object} configuration
+	 * @param {string} [configuration.snapshotEndpointUrl]
+	 */
+	constructor({snapshotEndpointUrl}) {
+		this.#snapshotEndpointUrl = snapshotEndpointUrl;
+	}
+
+	/**
+	 * SNAPSHOTs URL
+	 *
+	 * @public
+	 * @returns {string}
+	 */
+	getSnapshotEndpointUrl() {
+		return this.#snapshotEndpointUrl;
+	}
+
+	/**
+	 * @public
+	 * @returns {object} The configuration in a JSON format
+	 */
+	toJSON() {
+		return {
+			snapshotEndpointUrl: this.#snapshotEndpointUrl,
+		};
+	}
+}
+
+export default Configuration;
+
+/**
+ * Create Configuration from JSON file
+ *
+ * @public
+ * @static
+ * @param {string} [filePath="~/.ui5rc"] Path to configuration JSON file
+ * @returns {Promise<@ui5/project/config/Configuration>} Configuration instance
+ */
+export async function fromFile(filePath) {
```

Comment:
fromFile and saveConfig are currently not visible in JSDoc.
As the file represents the Configuration class (as it's written with uppercase "C") it would be best to only export the class.
What about adding those two functions as static methods to the class?

Reply:
Done